src/map/packets.h

```diff
@@ -2942,6 +2942,11 @@ packet(0x96e,-1,clif->ackmergeitems);
 	packet(0x08A4,36,clif->pStoragePassword,0);
 	packet(0x0363,8,clif->pDull); // CZ_JOIN_BATTLE_FIELD
 	packet(0x0436,4,clif->pDull); // CZ_GANGSI_RANK
+	packet(0x0988, 6);
```

Comment:
please not add packets in random place in packets.h.
Add existing one, or add if they missing in actual date where they was added to client.
I think all this packets already added. you only need find 0x098d and set function to clif->pClanMessage

If one of packets not added you can search this packets here: https://gitlab.com/4144/servergreps/tree/master/client_pre/packets/

Add packets in packets.h not mean change dates in clif.c
but clif.c date for packet usage must be >= date from packets.h